tests/ui/suggestions/issue-104086-suggest-let.stderr

```diff
@@ -7,7 +7,7 @@ LL |     x = x = x;
 help: you might have meant to introduce a new binding
    |
 LL |     let x = x = x;
-   |     +++
+   |     ~~~~~
```

Comment:
Could you keep the shorter span if the prefix *isn't* `let_`? It's more legible.

Reply:
sure